src/Language/Haskell/Liquid/Bare/OfType.hs

```diff
@@ -100,9 +102,10 @@ rtypePredBinds = map uPVar . ty_preds . toRTypeRep
 ofBRType :: (PPrint r, Reftable r)
          => (SourcePos -> RTAlias RTyVar SpecType -> [BRType r] -> r -> BareM (RRType r))
          -> (r -> BareM r)
+         -> (Expr -> BareM Expr)
          -> BRType r
          -> BareM (RRType r)
-ofBRType appRTAlias resolveReft
+ofBRType appRTAlias resolveReft resolveExpr
```

Comment:
Because `resolve` takes a `SourcePos` parameter which you don't always have. I looked into removing it a while back but concluded that that needed a small change to Fixpoint (which I could do, if it's wanted). A bit fuzzy on the details there at the moment.